web/src/main/java/org/cbioportal/web/mixin/SampleListMixin.java

```diff
@@ -0,0 +1,21 @@
+package org.cbioportal.web.mixin;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.cbioportal.model.CancerStudy;
+
+public class SampleListMixin {
+
+    @JsonIgnore
+    private Integer listId;
+    @JsonProperty("sampleListId")
```

Comment:
renaming of "stableId" in accordance with recently adopted standard on API return type field names